.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+default_install_hook_types: [pre-commit, post-checkout, pre-push]
+repos:
+  -   repo: local
+      hooks:
+        -   id: run lint
+            name: run lint
```

Comment:
As discussed, we can probably run lintfix now that precommit is used 🙂

Reply:
I think lintfix wont work since it modifies the file right.

Reply:
> modifies the file right.

That's a good thing, pre-commit will fail in this case so that we can verify and stage the modifications before recomitting

Many existing [hooks](https://pre-commit.com/hooks.html) rely on this behaviour.